### ISSUE TITLE
Plans on making nxxm open source?

### ISSUE THREAD

**Issue by biocomp** [nxxm/nxxm#1]
I could not find sources for the tool, and I'm guessing they were not released. Are you planning on doing so?

**Comment by daminetreg** [comment]
Dear @biocomp,

Thanks for looking at nxxm. :relaxed: 

Yes we will release the code fully opensource once we have grown, will surely happen by beginning of next year. 

We want to integrate it into visual studio code with a dedicated plugin first though and release it then Open Source.

We already released many components that we developed to build it : 
- [xxhr](https://nxxm.github.io/xxhr) : HTTP Requests cross-platform &amp; in the Browser
- [Github API implementation : nxxm/gh](https://github.com/nxxm/gh)
- [Command Line widgets nxxm/cli\_widgets](https://github.com/nxxm/cli_widgets)

All developers in the team are open source developers and we do love open source, we contributed to Boost.Fusion, Linux Kernel, Qt and other famous open source initiatives. 

We will still have the private Github repo / Github Enterprise connection as a paying feature though, but the code will be fully opensourced under appropriate license for reuse in other projects.

Cheers :grinning:
